staging/src/k8s.io/sample-apiserver/pkg/admission/plugin/banflunder/admission.go

```diff
@@ -44,7 +45,12 @@ func (d *disallowFlunder) Admit(a admission.Attributes) error {
 		return nil
 	}
 
-	flunderName := a.GetName()
+	metaAccessor, err := meta.Accessor(a.GetObject())
```

Comment:
as an example, it should illustrate how you would deal with both types of operations